plat/mediatek/mt8183/include/plat_debug.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef PLATFORM_DEBUG_H
+#define PLATFORM_DEBUG_H
+
+#define readl(addr)		mmio_read_32((addr))
+#define writel(addr, val)	mmio_write_32((addr), (val))
```

Comment:
Please use mmio_read/write_32() directly in your code so we always have the same accessors everyone knows across the codebase, don't define your own names for everything.